### DISCUSSION TITLE
MSF Support

### DISCUSSION THREAD

**Discussion by CoolHandSquid** [CoolHandSquid/TireFire#1] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
For Module/SmbVersionEnum.sh:

instead of the hardcoded ip address should be $1 // change line 15 from \\10.11.1.115 to \\$1 or \\$rhost 

consider adding this line:
           msfconsole -q -x "use auxiliary/scanner/smb/smb_version; set rhost $1; run; exit"

ran against windows 11 with no creds it gives:
SMB Version, Compression Capability, Encryption Capability, Guid, and Logon Server(Domain Name), and on earlier versions of Windows will give you the winders version as well.

Caveat: msf needs to be installed.

**Comment by CoolHandSquid** [comment]
Both of those changes make sense. I will investigate shortly; expect a push by Saturday Afternoon.

**Reply by CoolHandSquid** [reply]
Correction, Monday Afternoon.

**Reply by CoolHandSquid** [reply]
Updates Pushed! Thank you for the knowledge sharing and interaction. Expect some pretty major enhancements to TireFire in about 2 months!
